### ISSUE TITLE
Closing some old issues

### ISSUE THREAD

**Issue by rom1504** [lucidrains/DALLE-pytorch#262]
Looks like issues are accumulating. Some of them are already answered.
I think it's nice when there are fewer open issues.
Let's close them ?

If you're ok with that, I can post a "I think this can be closed" message in some of the issues.
What do you think?

**Comment by robvanvolt** [comment]
Agree! A good idea to stay focused on the current, important issues!

**Comment by afiaka87** [comment]
> Looks like issues are accumulating. Some of them are already answered.
> I think it's nice when there are fewer open issues.
> Let's close them ?
> 
> If you're ok with that, I can post a "I think this can be closed" message in some of the issues.
> What do you think?

I think a list of links posted here might be better for archival's sake. There's certainly a few issues which are definitely "solved" but not closed by the respective user. Then there's a slew of issues which are effectively "potentially important conversation" - some of these are pretty old, mostly surrounding the annoyingly coy nature of OpenAI. We can probably close most of those as well.

**Comment by rom1504** [comment]
Sounds good, I'll list some of them here

**Comment by afiaka87** [comment]
https://github.com/lucidrains/DALLE-pytorch/issues/234

**Comment by afiaka87** [comment]
@lucidrains ping

**Comment by afiaka87** [comment]
@lucidrains ping

**Comment by afiaka87** [comment]
@lucidrains ping

**Comment by afiaka87** [comment]
@lucidrains ping
